src/openssl/mod.rs

```diff
@@ -1010,12 +1004,11 @@ mod test {
         let public_key_bytes = public_key
             .to_bytes(&vrf.group, PointConversionForm::COMPRESSED, &mut vrf.bn_ctx)
             .unwrap();
-        println!("{:x?}", public_key_bytes);
```

Comment:
Wow, we missed that print! :worried: 

Now we should remove the unused variables `secret_key`, `public_key` and `public_key_bytes`.